scripts/envhook.py

```diff
@@ -41,6 +41,16 @@ def main(argv):
         else:
             raise
     if options.action == 'install':
+        try:
+            import sitecustomize
+        except ImportError:
+            pass
+        else:
+            if os.path.realpath(sitecustomize.__file__) == os.path.abspath(__file__):
+                raise RuntimeError('envhook is already installed')
```

Comment:
```suggestion
                pass
```

Reply:
I'm happy to change this, but isn't it better to be explicit?

Reply:
I see, this matches the pattern already set in the code.